packages/store/src/data-access/filesystem.ts

```diff
@@ -1,12 +1,13 @@
 import path from 'path';
-import fs, { readFileSync, writeFileSync, mkdirSync, FSWatcher, existsSync } from 'fs';
+import type { FSWatcher } from 'fs';
+import fs, { readFileSync, writeFileSync, mkdirSync, existsSync } from 'fs';
 import { plural } from 'pluralize';
-import { DataAccess, DataAccessConstructor } from '.';
-import { Logger } from '@sap-ux/logger';
+import type { DataAccess } from '.';
+import type { Logger } from '@sap-ux/logger';
 import { errorInstance, getFioriToolsDirectory } from '../utils';
-import { Entity } from '..';
-import { ServiceOptions } from '../types';
+import type { ServiceOptions } from '../types';
 import os from 'os';
+import type { Entity } from 'contants';
```

Comment:
`contants` ?

Reply:
Fixed in https://github.com/SAP/open-ux-tools/pull/374/commits/b43b771dd66aa53af1cc90b47e0ae4ede92c55aa